docs/getting-started.md

```diff
@@ -205,6 +205,15 @@ Now that you're setup, if you can run it on the command line, you can run it in
             - To start normal gameplay from Kickoff, press "Stop", then "Kickoff" for either team and then "Normal Start".
             - To learn more about how we coordinate different RefereeCommands to start special case gameplay behaviour (PenaltyKick, CornerKick, FreeKick), look at the [SSL rule documentation](https://ssl.robocup.org/rules/).
         - In addition, you can use ctrl-click to move the ball around in the Simulator, or try changing the Play Override on the Visualizer to select specific Plays!
+    
+    - If we want to run with one AI and / or Diagnostics
```

Comment:
This can be moved/combined with the doc lower down in this file:
https://github.com/UBC-Thunderbots/Software/blob/54418bde136b776d8a6c1bfcc916e49e0d314319/docs/getting-started.md?plain=1#L243-L250
In fact, the first point of part 2 of the doc can be removed now

Reply:
I refactored the doc a bit to explain the control mode menu a bit more and make it clearer, let me know what you think

Reply:
Looks good!